assembl/static2/js/app/pages/administration.jsx

```diff
@@ -530,6 +557,30 @@ export default compose(
       return !currentLocation.pathname.endsWith('discussion') || currentLocation.search !== '?section=3';
     }
   }),
+  graphql(LandingPageQuery, {
+    options: ({ i18n }) => ({
```

Comment:
maybe add a `skip:` param to avoid to fetch this data if we are not in landing page section?